addons/website_sale/views/templates.xml

```diff
@@ -2281,6 +2281,10 @@
         </xpath>
     </template>
 
+    <template id="all_products_link_name" name="All Products name">
+        <span placeholder="All products">All products</span>
```

Comment:
`placeholder` works on `span`?

Reply:
Okay, it's probably a feature of the website editor! Thanks for the update

Reply:
A *very bad* feature, it should be fixed at some point :wink: 
1. This is *not* valid HTML, it should just not be done (unfortunately... that's not the only one we have -_-)
2. That information has no value for the visitor (he does not see it), it should not be there (unfortunately this is done a lot for much other info, so at the moment, that part is ok, it's not really a big problem anyway).

Anyways, it can stay that way for now, we'll fix that editor feature everywhere at some point I guess (just `data-placeholder` at worse).